index.html

```diff
@@ -40,62 +40,67 @@
+          <li><a href="js/lesson7/tutorial.html">Lesson 7 - Introduction to Testing</a></li>
+          <li><a href="js/lesson8/tutorial.html">Lesson 8 - Building your own app</a></li>
+        </ul>
 
-          <h2>Ruby</h2>
+        <h2>Ruby</h2>
 
-          <p class="lead">If you are just getting started with Ruby, we recommend using <a href="http://nitrous.io">nitrous.io</a>, as setting up your local environment can be time consuming.</p>
+        <p class="lead">If you are just getting started with Ruby, we recommend using <a href="http://nitrous.io">nitrous.io</a>, as setting up your local environment can be time consuming.</p>
 
-          <ul>
-            <li><a href="ruby/lesson1/tutorial.html">Lesson 1 - Introduction to Ruby</a></li>
-            <li><a href="ruby/lesson2/tutorial.html">Lesson 2 - Ruby Basics</a></li>
-            <li><a href="ruby/lesson3/tutorial.html">Lesson 3 - Ruby Basics (part 2)</a></li>
-            <li><a href="ruby/lesson4/tutorial.html">Lesson 4 - Object Oriented Ruby and Inheritance</a></li>
-            <li><a href="ruby/lesson5/tutorial.html">Lesson 5 - Object Oriented Ruby and Inheritance (part 2)</a></li>
-          </ul>
+        <ul>
+          <li><a href="ruby/lesson1/tutorial.html">Lesson 1 - Introduction to Ruby</a></li>
+          <li><a href="ruby/lesson2/tutorial.html">Lesson 2 - Ruby Basics</a></li>
+          <li><a href="ruby/lesson3/tutorial.html">Lesson 3 - Ruby Basics (part 2)</a></li>
+          <li><a href="ruby/lesson4/tutorial.html">Lesson 4 - Object Oriented Ruby and Inheritance</a></li>
+          <li><a href="ruby/lesson5/tutorial.html">Lesson 5 - Object Oriented Ruby and Inheritance (part 2)</a></li>
+        </ul>
 
-          <h2>Python</h2>
+        <h2>Web development with Ruby & Sinatra</h2>
 
-          <ul>
-            <li><a href="python/lesson1/tutorial.html">Lesson 1 - Introduction to Python</a></li>
-            <li><a href="python/lesson2/tutorial.html">Lesson 2 - Playing with variables</a></li>
-          </ul>
-          </section>
-          <footer>
-          </footer>
-        </div>
-        <script src="javascripts/scale.fix.js"></script>
-        <script type="text/javascript">
-          var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
-          document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
-          </script>
-          <script type="text/javascript">
-            try {
-              var pageTracker = _gat._getTracker("UA-45098414-1");
-              pageTracker._trackPageview();
-            } catch(err) {}
-</script>
+        <ul>
+          <li><a href="sinatra/lesson1/tutorial.html">Lesson 1 - Introduction to Sinatra</a></li>
+        </ul>
+
+        <h2>Python</h2>
+
+        <ul>
+          <li><a href="python/lesson1/tutorial.html">Lesson 1 - Introduction to Python</a></li>
```

Comment:
the link to the second Python tutorial is missing ` <li><a href="python/lesson2/tutorial.html">Lesson 2 - Playing with variables</a></li>`